scripts/deploy.py

```diff
@@ -160,9 +160,31 @@ def preprocess_release():
         new_assets_file.write(content)
 
 
+def check_dev_dependencies_are_up_to_date():
+    """Runs start.sh to ensure that dev dependencies are up-to-date."""
+
+    cmd = ['bash', 'scripts/start.sh']
+    if os.path.exists('start_output.txt'):
+        os.remove('start_output.txt')
```

Comment:
Can you explain the uses of `start_output.txt`?

Reply:
Did you mean to remove dev_output.txt?

Reply:
Yes, my bad. Fixed now.